js/utils/backboneLocalStorage.js

```diff
@@ -0,0 +1,17 @@
+// https://github.com/jeromegn/Backbone.localStorage
+
+// We are wrapping the above Backbone localStorage adapter to slightly
+// change its behavior. By default, it overrides Backbone.sync globally,
+// forcing all models and collections to sync to localStorage
+// We are modifying it to leave the default sync in tact, and you can opt in on
+// a per model / collection basis by setting it's sync method to
+// Backbone.LocalStorage.sync. You also still need to set the localStorage attribute,
+// as described in the docs.
+
+import Backbone from 'backbone';
+import localStorageSync from 'backbone.localstorage'; // eslint-disable-line
```

Comment:
It's better to disable the specific eslint rule instead of the whole line, just in case there's also an unintentional error, and to make it clear which rule is being avoided.